### PR TITLE
Geoapify autocomplete

### PR DESCRIPTION
**Pull request by tz309806**
added POI search functionality

### CODE REVIEW

File: src/components/GeoapifySearchBoxComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// import React, {useState, useEffect} from 'react';
```
Comment (siddiqarz) [inline src/components/GeoapifySearchBoxComponent.tsx:1]: delete this file

File: src/navigation/RootNavigator.tsx [hunk 2]
```diff
@@ -98,10 +104,21 @@ const RootNavigator = () => {
           <Stack.Screen
             name="Tabs"
             component={TabNavigator}
-            options={{headerShown: false}}
+            options={{headerShown: false}} // Hide header for tabs
           />
           <Stack.Screen name="Home" component={Home} />
           <Stack.Screen name="Edit Profile" component={Profile} />
+          {/* Adding PlaceDetailsScreen and other stack navigations */}
+          {/*<Stack.Screen*/}
```
Comment (siddiqarz) [inline src/navigation/RootNavigator.tsx:112]: ?

File: src/services/GeoapifyFetchAutocompleteService.tsx [hunk 3]
```diff
@@ -0,0 +1,232 @@
+// import React, {useState, useRef} from 'react';
```
Comment (siddiqarz) [inline src/services/GeoapifyFetchAutocompleteService.tsx:1]: delete

File: android/build.gradle [hunk 4]
```diff
@@ -1,5 +1,19 @@
 buildscript {
     ext {
+        RNMapboxMapsImpl = "mapbox"
+        RNMapboxMapsVersion = '11.5.1'
+
+//        // Mapbox dependencies
```
Comment (siddiqarz) [inline android/build.gradle:6]: do we need this commented code?

File: src/components/GeoapifySearchBoxComponent.tsx (hunk 1 above)
Comment (tz309806) [reply src/components/GeoapifySearchBoxComponent.tsx]: Resolved

File: src/navigation/RootNavigator.tsx (hunk 2 above)
Comment (tz309806) [reply src/navigation/RootNavigator.tsx]: Resolved

File: src/services/GeoapifyFetchAutocompleteService.tsx (hunk 3 above)
Comment (tz309806) [reply src/services/GeoapifyFetchAutocompleteService.tsx]: Resolved

File: android/build.gradle (hunk 4 above)
Comment (tz309806) [reply android/build.gradle]: Resolved

**Pull request merged**